### DISCUSSION TITLE
Make button for POST request only

### DISCUSSION THREAD

**Discussion by ghost** [unpoly/unpoly#383] [Q&A] [answered]
Hi, I want to make a button with an href that sends a POST request without rendering the response on success, instead reloads the current page. Is it possible with tag attributes or do I need to use custom javascript?

Thanks

**Comment by triskweline** [comment, marked as answer]
I haven't tested by you can probably do this:

```html
<a href="/path" up-target=":none" up-on-loaded="up.reload()">Click me</a>
```

The `up.reload()` will reload the `<body>` element. If you want to reload the entire page you can also call `location.reload()`.

**Reply by ghost** [reply]
Yes, thank you this works, although I see it changes the url in location bar with the href path and renders the response for a fraction of second then redirects to the same page where I clicked the button.

**Comment by triskweline** [comment]
An `[up-navigate=false]` attribute will disable updates to the location bar and [other things](https://unpoly.com/navigation) you don't want when your link is not really a link.
